### PR TITLE
Update README.rst 

### PR DESCRIPTION
**Pull request by erogol**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -19,21 +19,53 @@
...
 
-**Documentation** for installation, usage, and training models are available on `stt.readthedocs.io <https://stt.readthedocs.io/>`_.
+**Coqui STT** (🐸STT) is a fast, open-source, multi-platform, deep-learning toolkit for training and deploying speech-to-text models. 🐸STT is battle tested in both production and research 🚀
 
-For the **latest release**, including pre-trained models and checkpoints, `see the latest release on GitHub <https://github.com/coqui-ai/STT/releases/latest>`_.
+🐸STT features
+---------------
 
-For contribution guidelines, see `CONTRIBUTING.rst <CONTRIBUTING.rst>`_.
+* High-quality pre-trained STT model.
```
Comment (JRMeyer) [inline README.rst:29]: s/model/models/

File: README.rst [hunk 2]
```diff
@@ -19,21 +19,53 @@
    
 |doc-img| |covenant-img| |gitter-img| |doi|
 
-**Coqui STT** (🐸STT) is an open-source deep-learning toolkit for training and deploying speech-to-text models. 🐸STT is battle tested in both production and research 🚀
+`👉 Subscribe to 🐸Coqui.ai Newsletter <https://coqui.ai/?subscription=true>`_
```
Comment (JRMeyer) [inline README.rst]: s/🐸Coqui.ai/🐸 Coqui's/

File: README.rst [hunk 3]
```diff
@@ -19,21 +19,53 @@
...
-For the **latest release**, including pre-trained models and checkpoints, `see the latest release on GitHub <https://github.com/coqui-ai/STT/releases/latest>`_.
+🐸STT features
+---------------
 
-For contribution guidelines, see `CONTRIBUTING.rst <CONTRIBUTING.rst>`_.
+* High-quality pre-trained STT model.
+* Eficient training pipeline with Multi-GPU support.
+* Sreaming inference.
+* Confidence score for each word.
+* Multiple possible result phrases each with an associated confidence.
```
Comment (JRMeyer) [inline README.rst:33]: s/possible result phrases/possible transcripts,/

s/confidence/confidence score/

File: README.rst [hunk 4]
```diff
@@ -19,21 +19,53 @@
...
+* Real-time inference.
+* Small-footprint acoustic model. 
+* Bindings for different programming languages.
```
Comment (JRMeyer) [inline README.rst:36]: s/different/various/

**Review by JRMeyer — CHANGES_REQUESTED** [review]
Can you link to `BUILDING.rst` from `index.rst` and link to `BUILD_DotNet.rst` from `BUILDING.rst`? 

I left single-line comments on a few nits.

File: README.rst (hunk 1 above)
Comment (erogol) [reply README.rst:29]: but we just have a model now

File: README.rst [hunk 5]
```diff
@@ -19,21 +19,53 @@
...
+**Coqui STT** (🐸STT) is a fast, open-source, multi-platform, deep-learning toolkit for training and deploying speech-to-text models. 🐸STT is battle tested in both production and research 🚀
 
-For the **latest release**, including pre-trained models and checkpoints, `see the latest release on GitHub <https://github.com/coqui-ai/STT/releases/latest>`_.
+🐸STT features
+---------------
 
-For contribution guidelines, see `CONTRIBUTING.rst <CONTRIBUTING.rst>`_.
+* High-quality pre-trained STT model.
+* Eficient training pipeline with Multi-GPU support.
+* Sreaming inference.
```
Comment (JRMeyer) [inline README.rst:31]: s/Sreaming/Streaming/

File: README.rst [hunk 6]
```diff
@@ -19,21 +19,53 @@
...
-**Documentation** for installation, usage, and training models are available on `stt.readthedocs.io <https://stt.readthedocs.io/>`_.
+**Coqui STT** (🐸STT) is a fast, open-source, multi-platform, deep-learning toolkit for training and deploying speech-to-text models. 🐸STT is battle tested in both production and research 🚀
 
-For the **latest release**, including pre-trained models and checkpoints, `see the latest release on GitHub <https://github.com/coqui-ai/STT/releases/latest>`_.
+🐸STT features
+---------------
 
-For contribution guidelines, see `CONTRIBUTING.rst <CONTRIBUTING.rst>`_.
+* High-quality pre-trained STT model.
+* Eficient training pipeline with Multi-GPU support.
```
Comment (JRMeyer) [inline README.rst:30]: s/Eficient/Efficient/

**Comment by erogol** [comment]
> Can you link to `BUILDING.rst` from `index.rst` and link to `BUILD_DotNet.rst` from `BUILDING.rst`?
> 
> I left single-line comments on a few nits.

I dont understand

File: README.rst (hunk 1 above)
Comment (reuben) [reply README.rst:29]: We have Mandarin but not a very good idea of how high-quality it is. Few test sets and not enough Mandarin knowledge

**Pull request merged**